interface/ceed-basis.c

```diff
@@ -1231,28 +1231,30 @@ int CeedBasisDestroy(CeedBasis *basis) {
 
   @ref Utility
 **/
-int CeedHaleTrefethenStripMap(CeedInt Q, CeedInt rho, CeedScalar *g, CeedScalar *g_prime) {
-  CeedScalar tau, d, C, PI2, PI = 4.0*atan(1.0);
+int CeedHaleTrefethenStripMap(CeedInt Q, CeedScalar rho, CeedScalar *g, CeedScalar *g_prime) {
```

Comment:
This still isn't a function $g(s)$ as we've discussed. You'll need to take `const CeedScalar *s` as an argument, then write `g[i]` as a function of `s[i]`.
![image](https://user-images.githubusercontent.com/3303/173156391-0e72ff96-08b5-4e2b-a0c7-dd1d9b47393c.png)